diagnostic.md

```diff
@@ -44,34 +49,45 @@ required for data to be returned to the client. (bullet points or ordered list)
 
 ```bash
 // your response here
+the GET request for `person/1` will be sent and 'activate' the 'person' controller.  the controller will recongizned that it is looking for the information of person id# 1 under the person datebase.  If the person model exist, it will send the request to 'person' model to retreive such information.  The model will then relay the information back the to controller. Then the controller will return the information back. If such model/database dont exist, controller will return an error.
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
 // your response here
+rails-api <folder name> ... (as ... for other options)
+i.e.: rails-api new blog_app --skip-javascript --skip-sprockets --skip-turbolinks --skip-test-unit --database=postgresql
+will start "new blog app" as a rail and with database in postgresql
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
 // your response here
+rails server
 ```
 
 What are the commands to drop, create and migrate a database? (3 bullet points)
 
 ```bash
 // your response here
+rake db:drop
+rake db:create
+rake db:migrate
+
 ```
 
 What is the command to scaffold a pet with a name and an age?
 
 ```bash
 // your response here
+rails-api g scaffold pet name:string age:integer
 ```
 
 List two advantages of using serializers? (2 bullet points)
 
 ```bash
 // your response here
```

Comment:
something like: to hide sensitive data, and make it easier to navigate JSON would suffice here